File: java/terreno.java
import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		float largura, comprimento, valor, area, preco;
		
		System.out.print("Digite a largura do terreno: ");
		largura = sc.nextFloat();

		System.out.print("Digite o comprimento do terreno: ");
		comprimento = sc.nextFloat();
		
		System.out.print("Digite o valor do metro quadrado: ");
		valor = sc.nextFloat();
		
		area = largura * comprimento;
		preco = valor * area;
		
		System.out.println("Area do terreno = " + String.format("%.2f", area) );
		System.out.println("Preco do terreno = " + String.format("%.2f", preco));
		
sc.close();
	}
}

File: c/terreno.c
#include <stdio.h>

int main()
{
    float largura, comprimento, valor_por_m, area, valor_terreno;

    printf("Digite a largura do terreno: ");
    scanf("%f", &largura);

    printf("Digite o comprimento do terreno: ");
    scanf("%f", &comprimento);

    printf("Digite o valor do metro quadrado: ");
    scanf("%f", &valor_por_m);

    area = comprimento * largura;
    valor_terreno = valor_por_m * area;


    printf("Area do terreno = %.2f\n", area);
    printf("Preco do terreno = %.2f\n", valor_terreno);

    return 0;
}
